src/main/scala/uk/gov/homeoffice/drt/repository/DepartureRepository.scala

```diff
@@ -78,7 +78,7 @@ class DepartureRepository[F[_] : Sync](val sessionPool: Resource[F, Session[F]])
     val a: F[List[DepartureTableData]] = selectDepartureTableData(arrivalTableData)
     val insertDepartureData: F[Option[DepartureTableData]] = a.map { b =>
```

Comment:
What is `insertDepartureData`? It sounds like something that needs to be inserted, but it's begin returned by a function called `ignoreScheduledDepartureIfExist`. It's not clear what this function is doing given it's name

Reply:
`insertDepartureData` is just simple insert of Departure Data and it is part of DepartureRepository class . May I should rename it to `ignoreScheduledDepartureIfExist` to `upsertScheduledDeparture`